### PR TITLE
Power and health bars added in UI and Db

### PR DESCRIPTION
**Pull request by aanchal07**
- Added power and health bars with icons in GameActivity and DressingRoom2 using IconRoundCornerProgressBar. 

- These include : healing, strength, invisibility, telepathy. They are randomly initialized.

- Linked these with database. When new user is there, these values are randomly initialized to value between 1 and 100 and stored in database. For the rest of the game, these values are accessed through the database.

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 1]
```diff
@@ -4,10 +4,14 @@
 import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 import android.widget.ImageButton;
 import android.widget.ImageView;
+import android.widget.Toast;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Remove this extra line

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 2]
```diff
@@ -228,7 +232,25 @@ public void onClick(View v) {
...
+                getmDbHandler().setStrength(strength);
+
+                r = new Random();
+                Integer invisibility = r.nextInt(101 - 1) + 1;
+                getmDbHandler().setInvisibility(invisibility);
+
+                r = new Random();
+                Integer telepathy = r.nextInt(101 - 1) + 1;
+                getmDbHandler().setTelepathy(telepathy);
+                Toast.makeText(AvatarRoomActivity.this, mDbHandler.getHealing() + " " + mDbHandler.getInvisibility() + " " + mDbHandler.getStrength() + " " + mDbHandler.getTelepathy(), Toast.LENGTH_LONG).show();
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Is there a need of the toast to the user? If only for debug purposes, log it.

File: PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java [hunk 3]
```diff
@@ -3,11 +3,18 @@
 import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
 import android.widget.ImageView;
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java]: Remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java [hunk 4]
```diff
@@ -3,11 +3,18 @@
...
 
+import com.akexorcist.roundcornerprogressbar.IconRoundCornerProgressBar;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java]: Remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java [hunk 5]
```diff
@@ -3,11 +3,18 @@
...
+import powerup.systers.com.db.DatabaseHandler;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java]: Remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java [hunk 6]
```diff
@@ -14,6 +14,9 @@
 import android.widget.ImageView;
 import android.widget.ListView;
 import android.widget.TextView;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: Remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java [hunk 7]
```diff
@@ -14,6 +14,9 @@
...
+import com.akexorcist.roundcornerprogressbar.IconRoundCornerProgressBar;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: Remove extra line

File: PowerUp/app/src/main/res/layout/pgame.xml [hunk 8]
```diff
@@ -10,17 +10,19 @@
 
     <TextView
         android:id="@+id/scenarioNameEditText"
+        android:hint="Scenario Name"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/pgame.xml]: Put in strings

**Comment by chhavip** [comment]
I have made minor comments on the coding style, fix them so I can review

**Comment by aanchal07** [comment]
@chhavip I have done the changes as you have pointed out.

**Comment by chhavip** [comment]
Will review soon!

**Pull request merged**